11ty/index.njk

```diff
@@ -16,9 +16,17 @@ layout: layouts/index.njk
       <div class="box">
```

Comment:
If this becomes too much text for the box, we have the latitude to make this `grid-column: span 2` if helpful.